<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowTransactions</class>
 <widget class="QDialog" name="ShowTransactions">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>517</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Transactions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../src/resource.qrc">
    <normaloff>:/icons/icons/view.png</normaloff>:/icons/icons/view.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*   style for the Root Widget  */
*{
background-color: rgb(126, 178, 235);
}


/* style for the groupRadio*/

#groupRadio{
	border: 1px solid #E72222;
	color: rgb(97, 33, 214);
}


#comboFrame{
	border: 1px solid #E72222;
}

#buttonFrame{
	border: 1px solid #E72222;
}


QComboBox{
	width: 200px;
	height: 30px;
	background-color: rgb(225, 225, 225);
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(32, 74, 135);
	border: 1px solid #264BF6;
	border-radius: 0px;
}


QPushButton{
	width: 100px;
	height: 30px;
	
	background-color: rgb(186, 189, 182);
	color: rgb(0, 0, 255);

	text-align: center;
	
	
	border: 1px solid #264BF6;
	border-radius: 5px;
   
}

QPushButton::hover{
	background-color: rgb(0, 0, 255);
    color: white;
	
	text-align: center;
		
	border: 1px solid #fff;
	border-radius: 5px;
}

QPushButton::disabled{
	color: rgb(0, 0, 0);
	background-color: rgb(136, 138, 133);
}


/*  style for the rightFrame  */
#tableView{
background-color: rgb(200, 231, 231);
}

#footerFrame{
background-color: rgb(200, 231, 231);
}

#footerFrame QLabel{
background-color: rgb(200, 231, 231);
}

#footerFrame QLineEdit{
background-color: rgb(245, 245, 245);
}

#editTotal{
font-size: 20px 
}

</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item alignment="Qt::AlignLeft">
    <widget class="QFrame" name="leftFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupRadio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Select Transaction Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioAll">
           <property name="text">
            <string>All</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioReceipts">
           <property name="text">
            <string>Receipt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioPayments">
           <property name="text">
            <string>Payment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="comboFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="labelAccountNumber">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Account Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboAccountNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>-----select account-----</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHeadName">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Transaction Head</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboHeadName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>-----select head-----</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTransDate">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Transaction Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboTransDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>-----Select  date-----</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="buttonSearch">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font/>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>&amp;Search</string>
           </property>
           <property name="icon">
            <iconset resource="../src/resource.qrc">
             <normaloff>:/icons/icons/view.png</normaloff>:/icons/icons/view.png</iconset>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QFrame" name="buttonFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonExport">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>&amp;Export as PDF</string>
           </property>
           <property name="icon">
            <iconset resource="../src/resource.qrc">
             <normaloff>:/icons/icons/show_users.png</normaloff>:/icons/icons/show_users.png</iconset>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBack">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>&amp;Back</string>
           </property>
           <property name="icon">
            <iconset resource="../src/resource.qrc">
             <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="rightFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="footerFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelAccount">
           <property name="text">
            <string>A/c No</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editAccount">
           <property name="text">
            <string>All</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHead">
           <property name="text">
            <string>Head</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editHead">
           <property name="text">
            <string>All</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDate">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editDate">
           <property name="text">
            <string>All</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTotal">
           <property name="text">
            <string>Total</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editTotal">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboAccountNumber</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onComboTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboHeadName</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onComboTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboTransDate</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onComboTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>ShowTransactions</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonExport</sender>
   <signal>clicked()</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onExportToPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onSelectTransactionType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioReceipts</sender>
   <signal>toggled(bool)</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onSelectTransactionType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioPayments</sender>
   <signal>toggled(bool)</signal>
   <receiver>ShowTransactions</receiver>
   <slot>onSelectTransactionType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onComboTextChanged()</slot>
  <slot>onSearch()</slot>
  <slot>onExportToPDF()</slot>
  <slot>onSelectTransactionType()</slot>
 </slots>
</ui>
